### PR TITLE
Templates added to .github

### LINKED ISSUE #77: Templates Required
Templates for New PR and Issues are required to allow easier and better descriptions of the same.

### PR DESCRIPTION
**Pull request by PrerakMathur20**
Fixes #77 
-

Pull Request Template has been added at `.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.md`
Issue Template has been added at `.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.md`

More templates can now also be added to the `.github` directory in the future as per convenience.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
```
Comment (maanas-talwar) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Could you please add the name of the test cases already in place as a checklist?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- [ ] Test B
+
+**Test Configuration**:
```
Comment (maanas-talwar) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: @PrerakMathur20 I believe these test configurations are not specific to this project and might not be required.

**Review by maanas-talwar — CHANGES_REQUESTED** [review]
Try adding the names of test cases already in place and looking into the test configurations.

**Comment by PrerakMathur20** [comment]
I have added all the required changes in my last commit, ready to be reviewed.

**Comment by maanas-talwar** [comment]
Try going through the testing demo video in the README.md for the list of unit tests.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (maanas-talwar) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Try going through the testing demo video in the readme for the list of unit tests present.

Comment (PrerakMathur20) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: I went through the Demo video, and I already have asked to run the `python3 app.py -v`  command. So clearly I'm either missing out on something or have misinterpreted this. So can you elaborate on the issue?

Comment (maanas-talwar) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: On running the unit tests, 5 different unit tests(testResults, testResultsJSON etc.) are run on the current state of the application.
Having a checklist for those 5 test cases is better.

**Comment by PrerakMathur20** [comment]
I hope these are the changes you were looking for. Kindly let me know if anything's wrong.

**Review by maanas-talwar — APPROVED** [review]
Looks good to me!

**Pull request merged**